cjworkbench/i18n/catalogs/extract.py

```diff
@@ -84,7 +84,7 @@ def extract_django(fileobj, keywords, comment_tags, options):
                     comments = []
                 comment = imatch.group(7)
                 if comment:
-                    comments.append(comment)
+                    comments.append(strip_quotes(comment))
```

Comment:
What does this do?

Reply:
At that point, the `comment` variable contains the comment as defined in the code, i.e. quoted, so we need to strip the quotes; I hadn't noticed it in the pull request that introduced the file. 

`strip_quotes` is defined at the top of the file as
```
def strip_quotes(s):
    if (s[0] == s[-1]) and s.startswith(("'", '"')):
        return s[1:-1]
    return s
```
and, like most of this file, it is copied from [https://github.com/python-babel/django-babel/blob/master/django_babel/extract.py].